examples/blender/blender.py

```diff
@@ -80,6 +83,8 @@ async def worker(ctx: WorkContext, tasks):
                 )
                 raise
 
+            s = new_script()  # reinitialize the script which we send to the engine
```

Comment:
Why do we need this?

Reply:
because otherwise, the function will try to use the old script that has already been sent to `process_batches`

Reply:
Got it, another option would be to have the one-off file upload be executed as a separate script.